canonical/ubuntu.com
# Fix infinite loops in /desktop and /cloud/public-cloud

My `redirects.yaml` created some infinite redirects. Here's my effort to fix them
## QA

Run the site and visit `/desktop` and `/cloud/public-cloud` and make sure they load and don't do an infinite redirect.
